src/cluster/cluster.cc

```diff
@@ -469,6 +469,52 @@ Status Cluster::GetClusterNodes(std::string *nodes_str) {
   return Status::OK();
 }
 
+Status Cluster::GetReplicas(const std::string &node_id, std::string *replicas_str) {
```

Comment:
Can use the StatusOr here:
```C++
StatusOr<std::string> Cluster::GetReplicas(const std::string &node_id) {

 ...
 return replicas_desc;
}
```

Reply:
[c9088c2](https://github.com/apache/kvrocks/pull/2244/commits/c9088c2dd2e7ea5ab206fbb2a707da4b2e18d167)